Doc/library/sys.rst

```diff
@@ -1154,7 +1154,9 @@ always available.
      string, which means the current working directory.
 
    To not prepend this potentially unsafe path, use the :option:`-P` command
-   line option or the :envvar:`PYTHONSAFEPATH` environment variable?
+   line option or the :envvar:`PYTHONSAFEPATH` environment variable. The
+   :option:`-p` option ignores the :option:`-P` command line option and the
+   :envvar:`PYTHONSAFEPATH` environment variable.
```

Comment:
Clarify semantics here; the `-p` option _causes_ the `-P` option to be ignored _by the interpreter_ (the actual subject), rather than being itself the subject of "ignore".

IMO, its clearer, simpler and more idiomatic to just phrase this as

```suggestion
   :option:`-p` option overrides the :option:`-P` command line option and the
   :envvar:`PYTHONSAFEPATH` environment variable.
```

but if you really want to use "ignore" as the verb, you could say

```suggestion
   :option:`-p` option causes the :option:`-P` command line option and the
   :envvar:`PYTHONSAFEPATH` environment variable to be ignored.
```